ner-search/README.md

```diff
@@ -0,0 +1,31 @@
+query:
+
+restaurants in San Francisco
+
+NER: [San Francisco] --> GEO
+
+matches:
+
+food in [San Francisco] -- higher match because same entity 
+
+restaurants in [Los Angeles] -- higher because same entity type
+
+bars in [New York] -- same as above
```

Comment:
This is arguable. You could also say: Should be removed, since wrong entity. Someone searching for restaurants in San Francisco most probably don't want to see any restaurant in their result set in New York.

Reply:
I think this could be achieved if you penalize `factor_type_match`